src/core/MIGRATION.md

```diff
@@ -643,22 +785,50 @@ With the legacy plugin system, extensions of core and other plugins are handled
 
 Each uiExport path is an entry file into one specific set of functionality provided by a client-side plugin. All webpack alias-based imports should be moved to these entry files, where they are appropriate. Moving a deeply nested webpack alias-based import in a plugin to one of the uiExport entry files might require some refactoring to ensure the dependency is now passed down to the appropriate place as function arguments instead of via import statements.
 
-### Switch to new platform services
+For the `plugins/` and `ui/` webpack aliases, you should be able to take advantage of the `legacy.ts` shim you created earlier. By placing these imports directly in your shim, you can pass the dependencies you need into your `Plugin.start` and `Plugin.setup` methods, from which point they can be passed down to the rest of your plugin's entry files.
```

Comment:
It might be worth mentioning, that this part is relevant for stateful things.

You may still have imports from `ui/` and `plugins/` if you need to import types and components.